tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import asyncio
+
+import pytest
+
+
+@pytest.fixture
```

Comment:
Do you need a new loop for each individual test or could you reuse the same loop.
If you can reuse the look please track it as ``scope='session'``

Reply:
We could, but I think it's better practice to create a new loop. If we wanted to reuse the loop we could call get_event_loop() instead of new_event_loop().